### PR TITLE
Feat/loss serialization

### PR DESCRIPTION
**Pull request by thib-s**
closing #26 , parameterized losses are now serializable objects inheriting tf.keras.Loss

### CODE REVIEW

**Review by franckma31 — APPROVED** [review]
OK 
Only need to replace KR() by KR in rst files

**Review by franckma31 — APPROVED** [review]
